DOC-600 Field level changes for release v.4.31 to v.4.37.0 (#65)
redpanda-data/rp-connect-docs: modules/components/partials/feature_stability_caution.adoc

ifndef::env-cloud[]
[CAUTION]
====

> This method is mostly stable. However, breaking changes could still be made to it outside of major version releases if a fundamental problem is found.

Reviewer: ```suggestion
Breaking changes to this method may occur outside of major version releases only if a fundamental issue is identified.
```

Reviewer: > This method is mostly stable.

This is too ambiguous and doesn't provide any useful information.

I added a suggestion but I still think it's weird. We're telling users that we aren't confident in the stability of the method and we won't necessarily follow the standard rules of semantic versioning if it breaks.

cc @Feediver1

Author: I've added the update for now. @Jeffail Is this just a legacy comment, or are breaking changes to Bloblang methods still released?

Reviewer 2: We haven't made any breaking changes in the last few years but it's worth keeping the messaging there for now. We'd need to review each experimental component to say for sure whether it's okay to mark as stable.

====
endif::[]
ifdef::env-cloud[]
[CAUTION]
====

> This method is mostly stable. However, breaking changes could still be made to it if a fundamental problem is found.

Reviewer: ```suggestion
Breaking changes to this method may occur if a fundamental issue is identified.
```
